src/_internals/itineraryLocation/index.tsx

```diff
@@ -63,6 +62,15 @@ const StyledItineraryLocation = styled(ItineraryLocation)`
     min-width: 0; /* reset flex default to allow shrinking below content width */
   }
 
+  & .kirk-itineraryLocation-connection {
+    display: flex;
+    align-items: center;
+    margin-top: 20px;
+  }
+  & .kirk-itineraryLocation-connection svg {
```

Comment:
maybe give a class for it instead of relying on the tag element

Reply:
Done